src/test/unit/controllers/EtDailyListController.test.ts

```diff
@@ -0,0 +1,83 @@
+import sinon from 'sinon';
+import fs from 'fs';
+import path from 'path';
+import {PublicationService} from '../../../main/service/publicationService';
+import {LocationService} from '../../../main/service/locationService';
+import {DataManipulationService} from '../../../main/service/dataManipulationService';
+import {Response} from 'express';
+import {mockRequest} from '../mocks/mockRequest';
+import EtDailyListController from '../../../main/controllers/EtDailyListController';
+
+const rawData = fs.readFileSync(path.resolve(__dirname, '../mocks/etDailyList.json'), 'utf-8');
+const listData = JSON.parse(rawData);
+
+const rawMetaData = fs.readFileSync(path.resolve(__dirname, '../mocks/returnedArtefacts.json'), 'utf-8');
+const metaData = JSON.parse(rawMetaData)[0];
+
+const rawDataCourt = fs.readFileSync(path.resolve(__dirname, '../mocks/courtAndHearings.json'), 'utf-8');
+const courtData = JSON.parse(rawDataCourt);
+
+const etDailyListController = new EtDailyListController();
+
+const etDailyListJsonStub = sinon.stub(PublicationService.prototype, 'getIndividualPublicationJson');
+const etDailyListMetaDataStub = sinon.stub(PublicationService.prototype, 'getIndividualPublicationMetadata');
+sinon.stub(LocationService.prototype, 'getLocationById').resolves(courtData[0]);
+sinon.stub(DataManipulationService.prototype, 'reshapeEtDailyListData').returns(listData);
+
+const artefactId = 'abc';
+
+etDailyListJsonStub.withArgs(artefactId).resolves(listData);
+etDailyListJsonStub.withArgs('').resolves([]);
+
+etDailyListMetaDataStub.withArgs(artefactId).resolves(metaData);
+etDailyListMetaDataStub.withArgs('').resolves([]);
+
+const i18n = {
+  'cop-daily-cause-list': {},
```

Comment:
Copying and pasting :) Should be et_daily_list

Reply:
Cheers